examples/text-generation/README.md

```diff
@@ -149,3 +149,18 @@ python run_generation.py \
 --prompt "Here is my prompt" \
 --peft_model trl-lib/llama-7b-se-rm-peft
 ```
+
+### Using growing bucket optimization
+With `--bucketsize`, instead of padding up the kvcache up to full size before starting, we grow the cache/input in multiples of bucketsize. This helps increase throughput and also reduce number of compilations if teh dataset has varying prompt lengths.
```

Comment:
```suggestion
With `--bucket_size`, instead of padding up the kv-cache up to full size before starting, we grow the cache/input in multiples of `bucket_size`. This helps increase throughput and also reduce number of compilations if the dataset has varying prompt lengths.
```